pseudoanonymize/common.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from dotenv import load_dotenv
+from joblib import Memory
+from openai import OpenAI
+
+load_dotenv()
+openai_api_key = os.getenv("OPENAI_API_KEY")
+client = OpenAI(api_key=openai_api_key)
+
+cachedir = "cache"
+memory = Memory(cachedir, verbose=0)
+
+
+def make_chat_completion(client, model, system_prompt, user_msg, temperature=0.1, context_length=4096):
```

Comment:
Yes, exactly. It seems like it's not possible to the cache make_chat_completion function directly because client is an object and its address changes on every run.